.github/workflows/main.yml

```diff
@@ -48,15 +48,6 @@ jobs:
         with:
```

Comment:
Does this whole ui-test section need removing?

Reply:
The tests are still there they are now part of the normal tests

Reply:
Oh yeah sorry, I was saying that so far you have just deleted a step in the ui-test section. I think the whole "ui-test:" object needs removing